website/src/docs/aws-s3-multipart.md

```diff
@@ -41,15 +41,15 @@ The `@uppy/aws-s3-multipart` plugin has the following configurable options:
 
 ### `limit: 6`
 
-The maximum amount of chunks to upload simultaneously. This affects [`prepareUploadParts()`](#prepareUploadParts-file-partData) as well; after the initial batch of `limit` parts is presigned, a minimum of `limit / 2` rounded up will be presigned at a time. You should set the limit carefully. Setting it to a value too high could cause issues where the presigned URLs begin to expire before the chunks they are for start uploading. Too low and you will end up with a lot of extra round trips to your server (or Companion) than necessary to presign URLs. If the default chunk size of 5MB is used, a `limit` between 5 and 6 is recommended.
+The maximum amount of chunks to upload simultaneously. You should set the limit carefully. Setting it to a value too high could cause issues where the presigned URLs begin to expire before the chunks they are for start uploading. Too low and you will end up with a lot of extra round trips to your server (or Companion) than necessary to presign URLs. If the default chunk size of 5MB is used, a `limit` between 5 and 6 is recommended.
 
 Because HTTP/1.1 limits the number of concurrent requests to one origin to 6, it’s recommended to always set a limit of 6 or smaller for all your uploads, or to not override the default.
 
-For example, with a 50MB file and a `limit` of 5 we end up with 10 chunks. 5 of these are presigned in one batch, then 3, then 2, for a total of 3 round trips to the server via [`prepareUploadParts()`](#prepareUploadParts-file-partData) and 10 requests sent to AWS via the presigned URLs generated.
+For example, with a 50MB file and a `limit` of 5 we end up with 10 chunks. 1 of these are presigned in one batch, then 1, then 1, etc. for a total of 10 round trips to the server via [`signPart()`][] plus one request through `createMultipartUpload` and one through `sendCompletionRequest` and 10 requests sent to AWS via the presigned URLs generated.
```

Comment:
```suggestion
For example, with a 50MB file and a `limit` of 5 we end up with 10 chunks. This means there will be 22 requests in total if you don't count the preflight (`OPTIONS`) requests: 1 creation request, 10 for signing parts, 10 for uploading the parts, and 1 to mark the upload as finished. 
```

Reply:
I don't know if we should keep this explanation at all if the goal is to remove that to use WebSocket in the future

Reply:
I don't think we reached consensus on the websocket idea yet. Let's not commit the docs to future plans that aren't certain yet

Reply:
I'm not saying we should commit to future plan, I'm saying maybe we are saying too much here and we could say less, no matter if WebSocket idea goes somewhere or not. If WebSocket thingy does become reality though, that sentence will be false, which is another reason to rethink that paragraph as it's not in our interest if users have wrong assumptions (also is it really useful for users to know how many requests are sent?)

Reply:
```suggestion
```